Integrations/integration-MaxMind_GeoIP2.yml

```diff
@@ -0,0 +1,293 @@
+        if 'registered_country' in res_json:
+            hr['ISO Code'] = res_json['registered_country']['iso_code']
+            maxmind_ec['ISO_Code'] = res_json['registered_country']['iso_code']
+            registration = res_json['registered_country']['names']['en']
+            hr['Registered Country'] = registration
+            maxmind_ec['RegisteredCountry'] = registration
+        if 'subdivisions' in res_json:
+            subs = res_json['subdivisions'][0]
+            hr['Subdivision'] = subs['names']['en']
+            maxmind_ec['Geo']['Subdivision'] = subs['names']['en']
+        if 'traits' in res_json:
+            traits = res_json['traits']
+            if 'user_type' in traits:
+                hr['User Type'] = traits['user_type']
+                maxmind_ec['UserType'] = traits['user_type']
+            if 'domain' in traits:
+                hr['Domain'] = traits['domain']
+                maxmind_ec['Domain'] = traits['domain']
+            if 'is_anonymous' in traits:
+                hr['Is Anonymous'] = traits['is_anonymous']
+                maxmind_ec['Anonymous'] = traits['is_anonymous']
+            if 'is_hosting_provider' in traits:
+                hr['Is Hosting Provider'] = traits['is_hosting_provider']
+                maxmind_ec['Host'] = traits['is_hosting_provider']
+            if 'is_tor_exit_node' in traits:
+                hr['Is TOR Exit Node'] = traits['is_tor_exit_node']
+                maxmind_ec['Tor'] = traits['is_tor_exit_node']
+            if 'autonomous_system_number' in traits:
+                hr['ASN'] = traits['autonomous_system_number']
+                ip_ec['ASN'] = traits['autonomous_system_number']
+                maxmind_ec['ASN'] = traits['autonomous_system_number']
+            if 'autonomous_system_organization' in traits:
+                hr['Organization'] = traits['autonomous_system_organization']
+                maxmind_ec['Organization'] = traits['autonomous_system_organization']
+            hr['IP'] = traits['ip_address']
+            ip_ec['Address'] = traits['ip_address']
+            maxmind_ec['Address'] = traits['ip_address']
+            if 'isp' in traits:
+                hr['ISP'] = traits['isp']
+                maxmind_ec['ISP'] = traits['isp']
+        return hr, ip_ec, maxmind_ec
+
+
+
+    ''' FUNCTIONS '''
+    def get_geo_ip(query):
+        raw = http_request(query)
+        res_json = raw.json()
+        return res_json
+
+
+    def geo_ip_command():
+        ip_query = demisto.args().get('ip')
+        res_json = get_geo_ip(ip_query)
+        hr, ip_ec, maxmind_ec = format_results(res_json)
+        ec = ({
+          'IP(val.Address && val.Address == obj.Address)': ip_ec,
+          'MaxMind(val.Address && val.Address == obj.Address)': maxmind_ec
+        })
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['markdown'],
+            'Contents': res_json,
```

Comment:
I remember you wasn't sure about this one, so try to use the `raw-respone=true` to check it out

Reply:
Maxmind, this works as expected. I think you're thinking about the alexa integration with the xml output.

Reply:
oh yeah, my bad